pkg/client/README.md

```diff
@@ -0,0 +1,132 @@
+# William
+
+This library aims to prevent writing to much code using william on your application.
+
+## Creating a service
+```go
+wi := william.New("URL OF WILLIAM", "ServiceName")
+```
+You can set the HTTP Client that will made the request also the tracing if anyone. (if not set nil)
+```go
+wi.SetClient(
```

Comment:
Does the lib provides a default client?

Reply:
The lib provide the default http client that don’t have a timeout and anyother security.

Reply:
WDYT about letting that explicit in the documentation?